src/main/java/com/chopshop166/chopshoplib/outputs/SwPIDSpeedController.java

```diff
@@ -18,6 +21,31 @@
     final private DoubleSupplier measurement;
     final private Watchdog dog = new Watchdog(1.0 / 50.0, this::calculate);
     private boolean enabled = true;
+    private IEncoder encoder = new MockEncoder();
+
+    public SwPIDSpeedController position(final SendableSpeedController motor, final PIDController pid,
+            final IEncoder encoder) {
```

Comment:
These can use the `IEncoder` in the SSC instead of requiring a separate one to be specified here.